src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,123 @@
+package duke;
+
+import duke.customexception.IllegalInputException;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Scanner;
+
+public class Duke {
+    public static void printLineSeparator() {
+        System.out.println("---------------------" +
+                "---------------------" +
+                "---------------------" +
+                "---------------------" +
+                "---------------------" +
+                "---------------------" +
+                "---------------------" +
+                "---------------------");
+    }
+
+    public static void printGreetings() {
+        String LOGO = " _           _        \n"
+                + "| |    _   _| | _____ \n"
+                + "| |   | | | | |/ / _ \\\n"
+                + "| |_  | |_| |   <  __/\n"
+                + "|____/ \\__,_|_|\\_\\___|\n";
+        System.out.println("Hello from\n" + LOGO);
+        System.out.println("Hello! I'm Luke");
+        System.out.println("What can I do for you?");
+        printLineSeparator();
+    }
+
+    public static boolean isTask(String line) throws IllegalInputException {
+        if (!(line.startsWith("todo") || line.startsWith("event") || line.startsWith("deadline"))) {
```

Comment:
perhaps move "todo" , "event" and  "deadline"  to constants?  you might need these commands elsewhere...